src/xrCore/Animation/BoneEditor.cpp

```diff
@@ -24,39 +22,17 @@ void SJointIKData::clamp_by_limits(Fvector& dest_xyz)
         dest_xyz.y = 0.f;
         clamp(dest_xyz.z, limits[1].limit.x, limits[1].limit.y);
         break;
-        /*
-            case jtWheelXZ:
-                clamp(dest_xyz.x,limits[0].limit.x,limits[0].limit.y);		dest_xyz.y=0;
-            break;
-            case jtWheelXY:
-                clamp(dest_xyz.x,limits[0].limit.x,limits[0].limit.y);		dest_xyz.z=0;
-            break;
-            case jtWheelYX:
-                clamp(dest_xyz.y,limits[1].limit.x,limits[1].limit.y);		dest_xyz.z=0;
-            break;
-            case jtWheelYZ:
-                clamp(dest_xyz.y,limits[1].limit.x,limits[1].limit.y);		dest_xyz.x=0;
-            break;
-            case jtWheelZX:
-                clamp(dest_xyz.z,limits[2].limit.x,limits[2].limit.y);		dest_xyz.y=0;
-            break;
-            case jtWheelZY:
-                clamp(dest_xyz.z,limits[2].limit.x,limits[2].limit.y);		dest_xyz.x=0;
-            break;
-        */
+    default:
+        break;
     }
 }
 
 void CBone::ShapeScale(const Fvector& _amount, bool /*parentCS = false*/)
 {
     switch (shape.type)
     {
-    case SBoneShape::stBox:
-    {
+    case SBoneShape::stBox: {
         Fvector amount = _amount;
-        //		Fmatrix _IT;_IT.invert(_LTransform());
-        //		_IT.transform_dir(amount,_amount);
-        //		if (parentCS) _IT.transform_dir(amount);
```

Comment:
Return deleted code **with** the original formatting.

Reply:
Restored.